src/org/exist/storage/BrokerWatchdog.java

```diff
@@ -8,10 +27,19 @@
 
 import org.exist.EXistException;
 
+/**
+ * Traces the lease of a Broker
+ *
+ * Note that tracing the stack is expensive
+ * this should only be done in debug mode by
+ * enabling the System Property {@link #TRACE_BROKERS_PROPERTY_NAME}
```

Comment:
Are there a few or many? I guess spending some time in documenting these is a good thing; I'll dive into this later today...